contrib/win32/openssh/OpenSSHUtils.psm1

```diff
@@ -273,28 +273,49 @@ function Repair-UserSshConfigPermission
 <#
     .Synopsis
     Repair-SSHFolderPermission
-    Repair the file owner and permission of ssh folder & any files inside it
+    Repair the folder owner and permission of ProgramData\ssh folder
 #>
 function Repair-SSHFolderPermission
 {
     [CmdletBinding(SupportsShouldProcess=$true, ConfirmImpact="High")]
     param (
         [parameter(Mandatory=$true)]
         [ValidateNotNullOrEmpty()]        
-        [string]$sshProgDataPath)
-
-    # SSH Folder - owner: System or Admins; full access: System, Admins; read or readandexecute/synchronize permissible: Authenticated Users
-    Repair-FilePermission -FilePath $sshProgDataPath -Owners $adminsSid, $systemSid -FullAccessNeeded $adminsSid,$systemSid -ReadAndExecuteAccessOK $authenticatedUserSid
-    # Files in SSH Folder (excluding private key files) 
-    # owner: System or Admins; full access: System, Admins; read/readandexecute/synchronize permissable: Authenticated Users
-    $privateKeyFiles = @("ssh_host_dsa_key", "ssh_host_ecdsa_key", "ssh_host_ed25519_key", "ssh_host_rsa_key")
-    Get-ChildItem -Path (Join-Path $sshProgDataPath '*') -Recurse -Exclude ($privateKeyFiles) -Force | ForEach-Object {
-        Repair-FilePermission -FilePath $_.FullName -Owners $adminsSid, $systemSid -FullAccessNeeded $adminsSid, $systemSid -ReadAndExecuteAccessOK $authenticatedUserSid
-    } 
-    # Private key files - owner: System or Admins; full access: System, Admins
-    Get-ChildItem -Path (Join-Path $sshProgDataPath '*') -Recurse -Include $privateKeyFiles -Force | ForEach-Object {
-        Repair-FilePermission -FilePath $_.FullName -Owners $adminsSid, $systemSid -FullAccessNeeded $systemSid, $adminsSid
-    }
+        [string]$FilePath)
+
+    Repair-FilePermission -Owners $adminsSid, $systemSid -FullAccessNeeded $adminsSid,$systemSid -ReadAndExecuteAccessOK $authenticatedUserSid @psBoundParameters
```

Comment:
-FilePath is mandatory parameter for Repair-FilePermission function? I don't see -FilePath here